shared/protocol/src/main/java/io/pravega/shared/NameUtils.java

```diff
@@ -68,11 +68,26 @@
      */
     private static final String EPOCH_DELIMITER = ".#epoch.";
 
+    /**
+     * Format for chunk name with segment name , epoch and offset.
+     */
+    private static final String CHUNK_NAME_FORMAT_WITH_EPOCH_OFFSET = "%s.E-%d-O-%d.%s";
+
     /**
      * Format for Container Metadata Segment name.
      */
     private static final String METADATA_SEGMENT_NAME_FORMAT = "_system/containers/metadata_%d";
 
+    /**
+     * Format for Storage Metadata Segment name.
+     */
+    private static final String STORAGE_METADATA_SEGMENT_NAME_FORMAT = "_system/containers/storage_metadata_%d";
+
+    /**
+     * Format for Container System Journal file name.
+     */
+    private static final String SYSJOURNAL_NAME_FORMAT = "_sysjournal.epoch%d.container%d.file%d";
```

Comment:
Why not put it in `_system/containers`? At least that way we'll keep all container-related stuff together.

Reply:
ok

Reply:
fixed.